### DISCUSSION TITLE
第1节课笔记(17班)

### DISCUSSION THREAD

**Discussion by Jianfeng777** [InternLM/Tutorial#77] [笔记]
请17班的同学上传笔记哦！

**Comment by AxrLH** [comment]
第一课笔记~
https://blog.csdn.net/Axr_Samuel/article/details/135373958?spm=1001.2014.3001.5502

**Comment by EthanHan1999** [comment]
第一节课笔记：https://zhuanlan.zhihu.com/p/675964568

**Comment by Jenniebymay** [comment]
第一节课笔记https://lyoufag8nd.feishu.cn/docx/R3ZcdxLD2oPSVbxDo4pc8Upen7d?from=from_copylink

**Comment by Alisa742** [comment]
第一节笔记：https://blog.csdn.net/weixin_53222965/article/details/135381680?spm=1001.2014.3001.5501

**Comment by Arvee-Li** [comment]
第一课笔记：[书生 第一课.md](https://github.com/InternLM/tutorial/files/13829273/default.md)

**Comment by zcg-joker** [comment]
[第一节课笔记](https://blog.csdn.net/m0_55764641/article/details/135373657)

**Comment by MarleneJiang** [comment]
第一节课笔记：https://juejin.cn/spost/7320079052013699098

**Comment by l241025097** [comment]
第一节课笔记：
![01《书生浦语大模型全链路开源体系》读书笔记](https://github.com/InternLM/tutorial/assets/31737569/6ea31a1d-2ae6-47eb-a1b0-20061573c4d9)

**Comment by wangncmyh** [comment]
第一节课笔记：
[Uploading 书生浦语第一节笔记.docx…]()

**Comment by Levitan911** [comment]
课时1笔记🔗：https://citrine-fibula-d4e.notion.site/1fbbde5107d54d3fb04b198ed913c9cf?pvs=4

**Comment by futtttttttttttt** [comment]
https://github.com/futtttttttttttt/InternLM/blob/master/InternLM/01%20%E7%AC%AC%E4%B8%80%E8%AF%BE/01%20%E7%AC%AC%E4%B8%80%E8%AF%BE.md

**Comment by langlibai66** [comment]
第一节课笔记：https://blog.csdn.net/weixin_74170225/article/details/135417955

**Comment by xaiojianjianbujiandan** [comment]
第一节课笔记：http://t.csdnimg.cn/9xsWk

**Comment by JACK626** [comment]
第一节课笔记：https://www.yuque.com/xiaotuanyue/mdr8d5/xo4sggiboz0rsa4i

**Comment by fine-lab** [comment]
第一节课笔记
https://blog.csdn.net/SmallZCan_t/article/details/135440789?spm=1001.2014.3001.5502

**Comment by popcornGit** [comment]
第一节课笔记
https://blog.csdn.net/weixin_38771145/article/details/135489882

**Comment by StarDuster** [comment]
第一节课笔记 https://gist.github.com/StarDuster/3ee6dea47febc24bc794a97763a91439
